dist/Dockerfile

```diff
@@ -1,18 +1,10 @@
-FROM centos
+FROM centos:7
 
 RUN yum -y groupinstall 'Development Tools'
 RUN yum -y install openssl-devel
 
-ADD https://static.rust-lang.org/dist/rust-1.29.0-x86_64-unknown-linux-gnu.tar.gz rust.tar.gz
-RUN tar -xf rust.tar.gz --strip 1
-RUN ./install.sh
+ENV RUST_LOG=actix_web=error
 
-RUN mkdir app
-WORKDIR app
-COPY . .
-RUN cargo build --release
+COPY target/x86_64-unknown-linux-musl/release/graph-builder target/x86_64-unknown-linux-musl/release/policy-engine /usr/bin/
 
-FROM centos
-ENV RUST_LOG=actix_web=error
 ENTRYPOINT ["/usr/bin/graph-builder"]
-COPY --from=0 /app/target/release/graph-builder /app/target/release/policy-engine /usr/bin/
```

Comment:
on our CI infra we don't yet have docker with multi-stage build feature so this will not work.